spec/models/essence_picture_spec.rb

```diff
@@ -68,33 +68,52 @@ module Alchemy
         end
       end
 
-      context 'with crop sizes set' do
-        before do
-          expect(essence).to receive(:crop_size).at_least(:once).and_return('200x200')
-          expect(essence).to receive(:crop_from).at_least(:once).and_return('10x10')
-        end
+      context 'when cropping is allowed' do
+        let(:options) { {crop: true} }
 
-        it "includes the crop sizes in the url." do
-          is_expected.to match(/200x200/)
-          is_expected.to match(/10x10/)
+        it 'converts the value `true` of the `crop` option into `crop` as part of the url' do
+          is_expected.to match /crop/
+          is_expected.not_to match /true/
         end
 
-        context 'but with crop sizes in the options' do
-          let(:options) { {crop_from: '30x30', crop_size: '75x75'} }
+        context 'and crop sizes are set' do
+          before do
+            expect(essence).to receive(:crop_size).at_least(:once).and_return('200x200')
+            expect(essence).to receive(:crop_from).at_least(:once).and_return('10x10')
+          end
+
+          it "includes the crop sizes in the url." do
+            is_expected.to match(/200x200/)
+            is_expected.to match(/10x10/)
+          end
+
+          context 'but with crop sizes in the options' do
+            before do
+              options.update({crop_from: '30x30', crop_size: '75x75'})
+            end
 
-          it "includes these crop sizes instead." do
-            is_expected.to match(/30x30/)
-            is_expected.to match(/75x75/)
+            it "includes these crop sizes instead." do
+              is_expected.to match(/30x30/)
+              is_expected.to match(/75x75/)
+            end
           end
         end
       end
 
-      context 'with crop true in the options' do
-        let(:options) { {crop: true} }
+      context 'when cropping is prohibited' do
+        let(:options) { {crop: false} } # This is default, but should make the test more understandable
```

Comment:
I changed it slightly by adding a new test to cover both scenarios:

* when we explicitly set `crop: false`
* when we do not set the crop key